src/formulations.jl

```diff
@@ -0,0 +1,286 @@
+
```

Comment:
This file is a buffer, where I move the implementation of the functions we have previously in `src/ExaPF.jl`. We should discuss what we need precisely to get rid of the dead code